source/plugin/basics/events.rst

```diff
@@ -171,6 +171,74 @@ Example: Listen for Custom Event
         event.getReceipient().sendMessage(ChatTypes.CHAT, "PM from " + senderName + ": " + event.getMessage());
     }
 
+Dynamic Registering and Unregistering of Event Handlers
+=======================================================
+
+Some Plugins may wish to dynamically register or unregister an event handler. In that case the event handler is not a method annotated with ``@Subscribe`` but a class implementing the ``EventHandler`` interface. This event handler can then be registered by calling the event managers ``register()`` method, which accepts a reference to the plugin, the ``Class`` of events handled and the handler himself.
+
+The exemplary creation and registration of a handler for an imaginary ``FLARDImplosionEvent`` is demonstrated below.
```

Comment:
You missed the FLARD. Also, 'exemplary' usually means a perfect or ideal example, when I believe saladoc just meant 'example'. It may become exemplary by being lauded...